compiler/build.gradle

```diff
@@ -100,7 +100,9 @@ model {
                 } else if (osdetector.os == "windows") {
                     linker.args "-static", "-lprotoc", "-lprotobuf", "-static-libgcc", "-static-libstdc++",
                             "-s"
-                } else {
+                }else if (System.getProperty('os.arch').contains('ppc64')){
```

Comment:
why not use `osdetector.arch` instead of `System.getProperty('os.arch')`? Also indent/format needs to be consistent

Reply:
@sanjaypujare I have made the change to use osdetector.arch as suggested. Thanks